pkgs/development/tools/electron/default.nix

```diff
@@ -47,7 +47,7 @@ let
 
       patchelf \
         --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-        --set-rpath "${atomEnv.libPath}:${gtk2}/lib:${at-spi2-atk}/lib:$out/lib/electron" \
+        --set-rpath "${atomEnv.libPath}:${stdenv.lib.makeLibraryPath [ gtk2 libuuid at-spi2-atk ]}:$out/lib/electron" \
```

Comment:
Does not electron use gtk3 nowadays?

Reply:
Don't see it being listed in the dependencies anymore so I guess not.